### ISSUE TITLE
Office plugin incorrectly identify Outlook add-in as PowerPoint add-in in MacOSX Outlook client

### ISSUE THREAD

**Issue by rdtsai** [MicrosoftDX/Vorlonjs#358]
Tested with the same Outlook add-in in both Outlook client and browser (OWA). Browser correctly identified this.officetype = e.OfficeTools.GetOfficeType(t.value.officeType) as "Outlook", Outlook client this.officetype returned as "PowerPoint".

The Office plugin for Outlook add-in did not work due to this incorrect behavior.

**Comment by meulta** [comment]
Ping @Mimetis ?

**Comment by Mimetis** [comment]
yeah ! I'm here :) Will take a look on this issue as soon as possible :)

**Comment by meulta** [comment]
Ping @Mimetis ?:)
